.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "max-len": ["error", {"code": 200, "ignoreComments": true}],
+    "no-unused-vars": "off",
+    "camelcase": "off",
+    "class-methods-use-this": "off"
```

Comment:
In the 3 offenses I found, all of them can be resolved by making those functions `static`.

Reply:
I cautioned against that because those basetypes were constructed directly from the mongoose documentation (which didn't use static). On second review, I think actually this is probably the way to go, and the mongoose documentation probably should of done it that way in the first place. Fixing now.